dashboard/test/models/blockly_test.rb

```diff
@@ -412,6 +429,52 @@ def assert_equal_xml(a, b)
     assert_not_equal localized_custom_block, translated_block
   end
 
+  test 'option that contains a period in the key is translated' do
+    test_locale = :"fr-ST"
```

Comment:
nit: we generally use `te-ST` for testing, to ensure that we're not using a real locale